lint381/c.py

```diff
@@ -11,9 +11,17 @@
 
 
 @linter.register
+@with_matched_tokens(start=match_regex("^sizeof$"), end=match_regex("^char$"))
+def prohibited_unsigned_char(source, *, match):
+    """sizeof char should be avoided as the C standard defines it to be a 1."""
```

Comment:
Use active voice (and also capitalize); something like "Flag sizeof(char) as it defined to be 1 and is redundant."